constants.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+    htmlSeparatorsDefault: {
+        rowStart: '<div class="table row">',
```

Comment:
This is going to be interesting, we can easily deduce an ending tag, do we need a rowStart and rowEnd? 

Could it simply be 

```
const tag = rowStart.match(/\<(.?+) /)
```

This will give the tag name even if it is made up.

Reply:
User can add a `<i class="fire"></i></div>` as end tag for extra markup in generate table.